src/target_intacct/payment_record.py

```diff
@@ -137,7 +137,8 @@ def payment_record_upload(intacct_client, config) -> None:
     input_value = get_input()
     
     if not input_value or not isinstance(input_value, list):
-        raise Exception(f"Invalid input data recieved. Input data={input_value}")
+        logger.warning(f"Invalid input data recieved. Stopping pipeline. Input data={input_value}")
+        exit()
```

Comment:
What happens if you were to `return` instead of `exit()`?

Reply:
That would work as well, is that better practice in this situation?

Reply:
Yes if ‘return’ will work it is preferred as it allows other parts of the program to still run and finish if needed. ‘exit’ is more restrictive and heavy handed.